packages/devp2p/src/types.ts

```diff
@@ -1,8 +1,66 @@
 import type { DPT } from './dpt/index.js'
+import type { ETH } from './protocol/eth.js'
+import type { LES } from './protocol/les.js'
 import type { Protocol } from './protocol/protocol.js'
+import type { SNAP } from './protocol/snap.js'
+import type { Peer } from './rlpx/peer.js'
 import type { Common } from '@ethereumjs/common'
+import type { NestedUint8Array } from '@ethereumjs/rlp'
 import type { Socket } from 'net'
 
+export interface RLPxEvent {
+  'peer:added': Peer
+  'peer:error': { peer: Peer; error: any }
+  'peer:removed': { peer: Peer; reason: any; disconnectWe: any }
```

Comment:
What is `disconnectWe`? Is there a more descriptive name for this field?

Reply:
It's just saying whether we "disconnected" the connection or the remote peer.  I'll add some comments on the types.